packages/blade/src/components/Indicator/__tests__/Indicator.native.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { Indicator } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
+describe('<Indicator />', () => {
+  it('should render', () => {
+    const { toJSON } = renderWithTheme(<Indicator accessibilityLabel="Success" />);
+
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render different variants for size and intent', () => {
```

Comment:
render all the size and intent perm and combination? i think it's ok for these kinds of tests to be verbose since they affect visual aspects of the component and also if one of the token changes we might not even discover if it affected indicator

Reply:
Added in https://github.com/razorpay/blade/pull/815/commits/5b465905cd5269dd300c467e55a465d2c4537851